chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -146,6 +146,7 @@ dataDir.migration.failure.partial.new                      = New directory: %S
 dataDir.migration.failure.partial.showDirectoriesAndQuit   = Show Directories and Quit
 dataDir.migration.failure.full.automatic.text1 = %S attempted to move your data directory to a new default location, but the migration could not be completed.
 dataDir.migration.failure.full.automatic.text2 = It is recommended that you close %S and move your data directory manually.
+dataDir.migration.failure.full.automatic.newDirOnDifferentDrive = Your current data directory is located on a different drive than the new default.
```

Comment:
Actually: "%S attempted to move your data directory to a new default location, but the old directory is on a different drive and cannot be migrated automatically."